src/snmp_core.cc

```diff
@@ -329,8 +329,7 @@ snmpClosePorts()
     }
     snmpIncomingConn = nullptr;
 
-    if (Comm::IsConnOpen(snmpOutgoingConn) && snmpIncomingConn != snmpOutgoingConn) {
-        // Perform OUT port closure so as not to step on IN port when sharing a conn.
+    if (Comm::IsConnOpen(snmpOutgoingConn)) {
```

Comment:
> PR description: and only close OUT socket if it is still open after closing IN.

Was not that already true before this change? In other words, AFAICT, before this PR changes, Squid closed OUT socket if it was still open after closing IN.

I am not against dropping `snmpIncomingConn != snmpOutgoingConn` condition, but current PR description does not seem to match current PR changes. Please rephrase. I can suggest "and simplify connection closing code".

Reply:
It looks correct to me. The text is describing the behaviour *after* PR is merged. Not what the PR changes.

Reply:
> It looks correct to me. The text is describing the behaviour _after_ PR is merged.

The same text is also describing the behaviour _before_ PR is merged. Please rephrase.